lib/Dialect/AIEVec/IR/AIE1/AIEVecAIE1Ops.cpp

```diff
@@ -0,0 +1,354 @@
+//===-- AIEVecAIE1Ops.cpp - MLIR AIE Vector Dialect Operations --*- C++ -*-===//
+//
+// This file is licensed under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// (c) Copyright 2024 Advanced Micro Devices, Inc. or its affiliates
+//
+//===----------------------------------------------------------------------===//
+// This file implements AIE1 vector op printing, pasing, and verification.
+//===----------------------------------------------------------------------===//
+
+#include "mlir/Dialect/LLVMIR/LLVMTypes.h"
+#include "mlir/IR/DialectImplementation.h"
+#include "mlir/IR/OpDefinition.h"
+#include "mlir/IR/TypeUtilities.h"
+#include "mlir/Transforms/FoldUtils.h"
+#include "llvm/ADT/TypeSwitch.h"
+
+#include "aie/Dialect/AIEVec/AIE1/IR/AIEVecAIE1Ops.h"
+#include "aie/Dialect/AIEVec/AIEVecUtils.h"
+
+using namespace llvm;
+using namespace mlir;
+
+// #include "aie/Dialect/AIEVec/IR/AIEVecEnums.cpp.inc"
+#include "aie/Dialect/AIEVec/AIE1/IR/AIEVecAIE1OpsDialect.cpp.inc"
+
+namespace xilinx::aievec::aie1 {
+
+//===----------------------------------------------------------------------===//
+// AIEVecAIE1Dialect
+//===----------------------------------------------------------------------===//
+
+void AIEVecAIE1Dialect::initialize() {
+  // registerTypes();
+  //   addAttributes<
+  // #define GET_ATTRDEF_LIST
+  // #include "aie/Dialect/AIEVec/IR/AIEVecAttributes.cpp.inc"
+  //       >();
```

Comment:
Do we need those comments?

Reply:
Will clean up the left over comments in next PR :)